Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang
+
+WORKDIR /go/src/gphotos-uploader-cli/
+COPY go.mod .
+COPY go.sum .
+RUN go mod download -x
+
+COPY . .
+RUN \
+  make test && \
+  make build && \
+  make clean
```

Comment:
Doesn't `make clean` remove the `gphotos-uploader-cli`?

Reply:
Forget it! I was testing it in a different version 😓

Reply:
For your own peace of mind, I already tested it specifically before submitting. 😄